config/pipeline.yaml

```diff
@@ -342,9 +361,91 @@ platforms:
 
   shell:
 
+  kontron-kswitch-d10-mmt-6g-2gs:
+    mach: at91
+    class: arm-dtb
+    dtb: 'lan966x-kontron-kswitch-d10-mmt-6g-2gs.dtb'
+    boot_method: uboot
+
+  kontron-kswitch-d10-mmt-8g:
+    mach: at91
+    class: arm-dtb
+    dtb: 'lan966x-kontron-kswitch-d10-mmt-8g.dtb'
+    boot_method: uboot
+
+  imx6dl-riotboard:
+    mach: imx
+    class: arm-dtb
+    boot_method: barebox
+    filters:
+      - blocklist:
+          defconfig:
+            - 'imx_v4_v5_defconfig'
+            - 'multi_v5_defconfig'
+            - 'allmodconfig'
+
+  imx6qp-wandboard-revd1:
+    mach: imx
+    class: arm-dtb
+    boot_method: barebox
+    flags: ['fastboot']
+    filters:
+      - blocklist:
+          defconfig:
+            - 'imx_v4_v5_defconfig'
+            - 'multi_v5_defconfig'
+            - 'allmodconfig'
+
+  dove-cubox:
+    mach: mvebu
+    class: arm-dtb
+    boot_method: barebox
+    filters:
+      - blocklist:
+          defconfig:
+            - 'allmodconfig'
+            - 'allnoconfig'
+            - 'tinyconfig'
+            - '-chromebook'
```

Comment:
This one will be tricky: I assume it means "exclude any defconfig containing `-chromebook`", for which we don't have an equivalent yet. Maybe having the following would do:
```
            fragments:
              - '!arm64-chromebook'
```

Not sure it would be usefull though, as the name implies we only have a fragment for arm64 chromebooks, but no other `arm*-chromebook` exists.

Reply:
well, I did as you mentioned for now